index/index.js

```diff
@@ -24,85 +24,177 @@ function weekDiff(start, end) {
+    for (var sectionIndex = 0; sectionIndex < 25; sectionIndex++) {
+      var value = sectionIndex + parseInt(event.params.offset,10);
+      if(value == sections.totalCount) {
+        break;
+      }
 
- // Invokes the banner lambda and gets the payload
-  const result = await invokeLambda(input);
+      // Gets all needed information for all the meeting times of a section
+      var meetingTimes = [];
+      var meetingLength = sections.data[sectionIndex].meetingsFaculty.length;
   
+      for (var meetingIndex = 0; meetingIndex < meetingLength; meetingIndex++) {
+        
+        // Calculates how many weeks for each meeting time
+        var start = sections.data[sectionIndex].meetingsFaculty[meetingIndex].meetingTime.startDate;
+        var end = sections.data[sectionIndex].meetingsFaculty[meetingIndex].meetingTime.endDate;
+        var diff = weekDiff(start, end);
+        
+        // Finds campus code for each meeting time
+        var campus;
 
-  // Loops through the payload (Need to change the for loop values)
-  for (var index = parseInt(event.params.offset); index < parseInt(event.params.pageSize); index++) {
-    if(index == result.totalCount) {
-      break;
-    }
-    var len = result.data[index].meetingsFaculty.length;
-    var times = [];
-
-    for (var index2 = 0; index2 < len; index2++) {
-      // Calculates how many weeks for each course
-      var start = result.data[index].meetingsFaculty[index2].meetingTime.startDate;
-      var end = result.data[index].meetingsFaculty[index2].meetingTime.endDate;
-      var diff = weekDiff(start, end);
-      
-      // Gets faculty information for each course
-      var staff = [];
-      var len2 = result.data[index].faculty.length;
-      for(var j = 0; j < len2; j++) {
-        staff.push(result.data[index].faculty[j].displayName);
+        var campusesLength = campuses.length;
+        for(var index = 0; index < campusesLength; index++) {
+          if(String(sections.data[sectionIndex].campusDescription).localeCompare(String(campuses[index].description)) == 0) {
+            campus = campuses[index].code;
+          }
+       }
+          
+        // Gets faculty information for each meeting time
+        var staff = [];
+        var facultyLength = sections.data[sectionIndex].faculty.length;
+
+        for(index = 0; index < facultyLength; index++) {
+          staff.push(sections.data[sectionIndex].faculty[index].displayName);
+        }
+        
+        // Gets start and end time for each meeting time
+        let startTime;
+        let finishTime;
+        
+        try {
+          startTime = Number(sections.data[sectionIndex].meetingsFaculty[meetingIndex].meetingTime.beginTime);
+        } catch (err) {
+          startTime = 'N/A';
+        }
+        
+         try {
+          finishTime = Number(sections.data[sectionIndex].meetingsFaculty[meetingIndex].meetingTime.endTime);
+        } catch (err) {
+          finishTime = 'N/A';
+        }
```

Comment:
If there is an error, you should set the start/end time to a number that would indicate that, such as -1. This way the data type is consistent.